contracts/test/accounts/Account.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+
+import { IBasicFDT }    from "../../interfaces/IBasicFDT.sol";
+import { IExtendedFDT } from "../../interfaces/IExtendedFDT.sol";
+
+
+contract Account {
+
+
+    /******************/
+    /*** Basic FDT ***/
+    /*****************/
```

Comment:
These should all be grouped into Direct Functions and Try Functions, there should only be two comment sections